src/styles/blocks/_main-content.scss

```diff
@@ -0,0 +1,26 @@
+@use "../utils/mixins" as *;
+
+.main-content {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  width: 1024px;
+
+  @include large-up {
+    padding: 50px 128px;
+    width: 1200px;
+  }
```

Comment:
This part is not needed, as with width growth you also raise left/right padding so you can stay with default styles for this main-content